core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -457,6 +457,9 @@ public enum SqlKind {
   /** {@code TIME_SUB} function (BigQuery). */
   TIME_SUB,
 
+  /** {@code TIMESTAMP} function (BigQuery). */
+  TIMESTAMP,
```

Comment:
not a good name, since TIMESTAMP is a type

Reply:
Done, didn't need to add that anyways I don't think.